compliance/locker.py

```diff
@@ -514,14 +527,19 @@ def load_content(self, evidence, ignore_ttl=False, evidence_dt=None):
                     root.extend(parse_dot_key(data, evidence.part_root))
                 else:
                     root.extend(data)
-            evidence.set_content(format_json(content))
+            evidence.set_content(format_json(content), sign=False)
         else:
             evidence.set_content(
                 self._get_file_content(
                     evidence.path,
                     evidence_dt,
                     getattr(evidence, 'binary_content', False)
-                )
+                ),
+                sign=False
+            )
+        if not evidence.verify(self):
+            raise UnverifiedEvidenceError(
+                f'Evidence {evidence.path} could not be verified.', evidence
             )
```

Comment:
Could be nice to split the logic in `verify()` into `is_signed()` and `verify_signature()`. Also makes it more explicit, as you could imagine various things you could verify about a piece of evidence.

Could also make the error message more specific:
```suggestion
        if evidence.is_signed(self):
            if not evidence.verify_signature(self):
                raise UnverifiedEvidenceError(
                    f'Evidence {evidence.path} is signed, but the signature could not be verified.', evidence
                )
```